api/CHANGELOG.md

```diff
@@ -40,6 +40,23 @@
 
 * FIXED: Remove optional parent_context from in_span
 
+### v1.0.0 / 2021-09-29
+
+* BREAKING CHANGE: Remove optional parent_context from in_span 
+* BREAKING CHANGE: Refactor Baggage to remove Noop* 
+* BREAKING CHANGE: Total order constraint on span.status= 
+
+* ADDED: Add Tracer.non_recording_span to API 
+* ADDED: Make API's NoopTextMapPropagator private 
+* FIXED: Remove optional parent_context from in_span 
+* FIXED: Reduce span allocation in API 
+* FIXED: Refactor Baggage to remove Noop* 
+* FIXED: Total order constraint on span.status= 
+* FIXED: Return early if carrier is nil 
+* FIXED: Update context to match spec 
+* FIXED: Return the original context if the baggage header value is empty 
+* DOCS: Update docs to rely more on environment variable configuration 
```

Comment:
This is interesting - the changeless includes all the things from the RCs. Is this what we want? I feel like it doesn't correctly represent the release history. This should be pulled up to the top of the file, and I think there should be "No significant changes".

Reply:
Yeah it is auto generated, about to push an update to the changelog.